test/Flick/AnimatedCarousalPage.xaml

```diff
@@ -11,29 +11,41 @@
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
     <Grid Margin="10" KeyDown="Grid_KeyDown">
-        <ScrollViewer x:Name="sv" Height="300"
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="9*"/>
+            <RowDefinition />
+        </Grid.RowDefinitions>
+        <ScrollViewer x:Name="sv"
                       HorizontalScrollBarVisibility="Hidden"
                       VerticalScrollBarVisibility="Disabled"
                       HorizontalScrollMode="Enabled"
                       VerticalScrollMode="Disabled"
                       HorizontalSnapPointsAlignment="Center"
                       HorizontalSnapPointsType="Mandatory"
-                      Grid.Row="1">
-            <local:SnappPointForwardingRepeater x:Name="repeater" HorizontalCacheLength="0" VerticalCacheLength="0" ElementPrepared="OnElementPrepared">
+                      VerticalAlignment="Center"
+                      Grid.Row="1"
+                      ViewChanged="OnScrollViewerViewChanged"
+                      ViewChanging="OnScrollViewerViewChanging"
+                      Tapped="Sv_Tapped">
+            <local:SnappPointForwardingRepeater x:Name="repeater" HorizontalCacheLength="0.5" VerticalCacheLength="0" ElementPrepared="OnElementPrepared">
                 <mux:ItemsRepeater.Layout>
-                    <local:VirtualizingUniformCarousalStackLayout x:Name="layout" ItemWidth="300" ItemHeight="350" Spacing="10"/>
+                    <local:VirtualizingUniformCarousalStackLayout x:Name="layout" ItemWidth="450" ItemHeight="525" Spacing="50"/>
```

Comment:
>ItemWidth="450" ItemHeight="525" [](start = 82, length = 32)

perhaps for a separate PR, we could figure out the size based on measuring one of the items. I can show you how we do it in UniformWrapGrid. That way if no ItemWidth/height is provided, we can figure out for the size of one of the items.

Reply:
Sounds good.